WEB-INTEGRATION.md

```diff
@@ -357,6 +373,10 @@ In general, the context that should be used is the one that matches the data
 flow through the algorithms ([see the section on implicit propagation
 below](#implicit-context-propagation)).
 
+<!-- Streams are laregely defined on top of promises, and can be easily
+reimplemented in userland by copying the spec step-by-step. Likely this
+described propagation already properly comes out of that? -->
```

Comment:
Piping is left up to implementations, with constraints limiting the allowed behavior (see [`ReadableStreamPipeTo`](https://streams.spec.whatwg.org/#readable-stream-pipe-to), step 15). In particular, this step says that this must happen "in parallel, but not really" (see whatwg/streams#905).

But we probably should just add a constraint saying that if the implementation of this step directly results in calls to the underlying sink/source callbacks, that those calls must happen in the same context that `ReadableStreamPipeTo` was called in.